blog/green/20240621-greenIT-introduction-1/index.md

```diff
@@ -0,0 +1,90 @@
+
+!["The wheel of metal companionality](roue-metaux.jpeg)
+
+Pour extraire des métaux de la roche, il y a des procédés mécaniques, thermiques, chimiques pour par exemple partir de la roche et obtenir des catodes de cuivres. Tout celà avec une consommation d'eau et d'énergie phénoménale.
+
+![Exemple extraction issu de https://www.systext.org/sites/default/files/RP_SystExt_Controverses-Mine_VOLET-1_Nov2021_maj.pdf](exemple-cuivre.png)
+
+## Il faut creuser un gros trou ?
+
+Vous avez peut-être déjà vu les [photographies de Dillon Marsh](https://dillonmarsh.com/fwiw.html) qui montre "à l'échelle" la taille des mines, versus ce qui en est extrait.
+
+Ci-dessous, on peut se rendre compte, avec la représentation de la tour Eiffel, de la taille d'un site (extraction + exploitation + emprise en surface des déchets) pour la "mine de cuivre" de Palabora
+![palabora-Afrique-Sud_Dillon-Marsh.jpg Illustration de la mine de Palabora avec taille du métal extrait au global versus taille de l'exploitation minière](palabora-Afrique-Sud_Dillon-Marsh.jpg)
+
+Située en Afrique du Sud, cette mine a été exploitée pendant presque 40 ans [plus de détail sur cette mine avec ce lien](https://planet-terre.ens-lyon.fr/ressource/Img784-2023-06-05.xml).
+D'autres exemples de mines gigantesques existent (cette fois c'est du charbon, mais ça symbolise bien l'extraction minière), plus proche de nous, comme la [mine de Hambach en Allemagne](https://fr.wikipedia.org/wiki/Mine_de_Hambach#)
+
+![Johannes Fasolt](surface_Mining_Hambach.jpg)
+
+## Quelles conséquences l'exploitation minière ?
+
+La conséquence principale de cette exploitation est bien entendu d'approvisionner toutes nos industries : transports, énergie, armement, agriculture... enfin tout ce qui nécessite aujourd'hui des métaux, c'est à dire à peu prés tout, dans sa composition,sa construction ou sa logistique. Faites l'exercice : "Quel artefact utilisez-vous dans votre quotidien qui n’est pas dépendant de l’industrie minière dans sa fabrication, sa vente, son transport ?"
+
+Conséquences écologiques.
+conséquences politiques et sociales, parler de fairphone et des mines de Cobalt...
+
+## Quelles solutions pour diminuer l'impact ?
+Ne soyons pas pessimistes sur la capacité de l'Homme à trouver des solutions, mais nous passerons forcément à une diminution de l'usage des métaux. D'abord, parce que la teneur en métaux baissent dans les exploitations (même si on trouvera toujours de nouvelles mines) et ensuite parceque ça coutera toujours plus d'énergie, qui est elle aussi n'est pas inépuisable ?
+
+Alors à l'échelle du numérique, que pouvons-nous faire ? Se poser les bonnes questions sur les usages toujours plus gourmands en ressources, concevoir mieux pour utiliser plus longtemps.
```

Comment:
je rajouterai un dernier paragraphe pour annoncer les prochains articles orientés eco conception it et autres non ?